modules/node_groups/locals.tf

```diff
@@ -31,13 +31,34 @@ locals {
     v,
   ) if var.create_eks }
 
-  node_groups_names = { for k, v in local.node_groups_expanded : k => lookup(
-    v,
-    "name",
-    lookup(
-      v,
-      "name_prefix",
-      join("-", [var.cluster_name, k])
-    )
-  ) }
+  # This node_groups_names construct is a consequence of not explicitly
```

Comment:
I think that this comment is also not required as if somebody will try to understand this then will need analyse that conditional